custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -314,14 +314,20 @@ def _reg_arg_url(*parsers):
 
     @staticmethod
     def _reg_arg_language(*parsers):
-        langs = [e.value for e in RunLanguage]
-        langs.remove(RunLanguage.JAVA.value)
         for parser in parsers:
+            langs = [e.value for e in RunLanguage]
+            prog_name_lst = CMRunnerArgsRegistry._tokenize_parser_prog(parser)
```

Comment:
can you explain briefly what does this logic below means?

Reply:
```
@staticmethod
    def _tokenize_parser_prog(parser):
        # example:
        # - for score_parser prog is "drum score"
        # - for new_model_parser prog is "drum new model"
        return parser.prog.split(" ")
```
So i split prog by tokens to be able to customize arg's behavior for different subcommands.

Reply:
I see, but what about "if its a new model, remove java from langs"?

Reply:
so from DMs FYI, this means that this logic doesnt support creating a template for java for now.
As well we figured it out that we might start doing so for mojo/pojo.

Reply:
https://datarobot.atlassian.net/browse/RAPTOR-3330